website/pages/en/developing/creating-a-subgraph.mdx

```diff
@@ -410,22 +410,28 @@ calls:
 The ABI file(s) must match your contract(s). There are a few ways to obtain ABI files:
 
 - If you are building your own project, you will likely have access to your most current ABIs.
-- If you are building a subgraph for a public project, you can download that project to your computer and get the ABI by using [`truffle compile`](https://truffleframework.com/docs/truffle/overview) or using solc to compile.
+- If you are building a subgraph for a public project, you can download that project to your computer and get the ABI by using [`npx hardhat compile`](https://hardhat.org/hardhat-runner/docs/guides/compile-contracts#compiling-your-contracts) or using solc to compile.
```

Comment:
```suggestion
- If you are building a subgraph for a public project, you can download that project to your computer and get the ABI by using [`npx hardhat compile`](https://hardhat.org/hardhat-runner/docs/guides/compile-contracts#compiling-your-contracts) or using `solc` to compile.
```
Not related to your change, but since it's on the same line, I thought why not :P